.venv/project/ecommerce/api/itembrowsing.py

```diff
@@ -1,11 +1,15 @@
 from firebase_admin import firestore
+from difflib import SequenceMatcher
+from ..models.items import Item
+
+
 
 db = firestore.client()
 
 #reference to items collections
 items_ref=db.collection(u'items')
 
-def addItems(self,name="",sellerID=0,photo="",price=0,description="",weight=0,rating=0, score=0,sales=False,category=None):
+def addItems(name="",sellerID=0,photo="",price=0,description="",weight=0, score=0,sales=False,category=None,amountInStock=0,):
```

Comment:
I don't think we should include stock since we don't have any logic to deal with it anywhere else in the application

Reply:
Removed it.